openlibrary/plugins/recaptcha/recaptcha.py

```diff
@@ -35,4 +35,11 @@ def validate(self, value=None):
             )
             return True
 
-        return r.json().get('success', '')
+        data = r.json()
+        if not data.get('success', False) and 'error-codes' in data:
+            from openlibrary.plugins.openlibrary.sentry import sentry
+
+            if sentry and sentry.enabled:
+                sentry.capture_message(f"Recaptcha Error: {data['error-codes']}")
+
```

Comment:
logged exceptions are captured in sentry, so we should be able to do:

```suggestion
        if not data.get('success', False) and 'error-codes' in data:
            logging.getLogger("openlibrary").exception(f"Recaptcha Error: {data['error-codes']}")
```

I believe.